exercises/practice/simple-linked-list/test/linked_list_test.exs

```diff
@@ -118,11 +118,22 @@ defmodule LinkedListTest do
   end
 
   @tag :pending
-  test "to_list/1 of list of 2 datum" do
+  test "to_list/1 of list of 2 datum, keeping order" do
     list = LinkedList.from_list([:mon, :tues])
     assert LinkedList.to_list(list) == [:mon, :tues]
   end
 
+  @tag :pending
+  test "from_list/1 and successive push/2 of a list result in reversed order" do
+    list = [:mon, :tues]
+
+    ll_via_from_list = LinkedList.from_list(list)
+    ll_via_pushed = Enum.reduce(list, LinkedList.new(), &LinkedList.push(&2, &1))
```

Comment:
Variable names with `l` are hard to read because they look like `1`. 
Maybe better names could be `linked_list` and `pushed_list`?

Reply:
Done! Thx!